packages/salesforcedx-vscode-visualforce/src/extension.ts

```diff
@@ -106,79 +106,86 @@ export async function activate(context: ExtensionContext) {
 
   let disposable = client.start();
   toDispose.push(disposable);
-  client.onReady().then(() => {
-    disposable = languages.registerColorProvider(documentSelector, {
-      provideDocumentColors(
-        document: TextDocument
-      ): Thenable<ColorInformation[]> {
-        const params: DocumentColorParams = {
-          textDocument: client.code2ProtocolConverter.asTextDocumentIdentifier(
-            document
-          )
-        };
-        return client
-          .sendRequest(DocumentColorRequest.type, params)
-          .then(symbols => {
-            return symbols.map(symbol => {
-              const range = client.protocol2CodeConverter.asRange(symbol.range);
-              const color = new Color(
-                symbol.color.red,
-                symbol.color.green,
-                symbol.color.blue,
-                symbol.color.alpha
-              );
-              return new ColorInformation(range, color);
-            });
-          });
-      },
-      provideColorPresentations(
-        document: TextDocument,
-        colorInfo: ColorInformation
-      ): Thenable<ColorPresentation[]> {
-        const params: ColorPresentationParams = {
-          textDocument: client.code2ProtocolConverter.asTextDocumentIdentifier(
-            document
-          ),
-          colorInfo: {
-            range: client.code2ProtocolConverter.asRange(colorInfo.range),
-            color: colorInfo.color
-          }
-        };
-        return client
-          .sendRequest(ColorPresentationRequest.type, params)
-          .then(presentations => {
-            return presentations.map(p => {
-              const presentation = new ColorPresentation(p.label);
-              presentation.textEdit =
-                p.textEdit &&
-                client.protocol2CodeConverter.asTextEdit(p.textEdit);
-              presentation.additionalTextEdits =
-                p.additionalTextEdits &&
-                client.protocol2CodeConverter.asTextEdits(
-                  p.additionalTextEdits
+  client
```

Comment:
The auto formatter is driving me crazy with this one. I'll get this settled in another commit.